src/app/component/user/auth/sign-up/sign-up.component.css

```diff
@@ -1,21 +1,305 @@
-.sign-up {
+.grey-bg {
```

Comment:
how about using some general style file for sign-in and sign-up component to avoid duplication code?!

Reply:
some parts of css-code was moved to main css-file (for forms only) , and  amin css file was included to another component
new pull-request for this fixes

Reply:
https://github.com/ita-social-projects/GreenCityClient/pull/149